### PR TITLE
Define and activate AWS 9.3.3 release

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/11379

### CODE REVIEW

File: aws.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
   name: v11.3.2
 spec:
   state: active
-  date: 2020-06-03T15:00:00Z
+  date: 2020-06-04T15:00:00Z
```
Comment (sslavic) [inline aws.yaml:9]: Updated 11.3.2 release date to the day it was actually released.

File: aws.yaml [hunk 2]
```diff
@@ -421,9 +421,61 @@ kind: Release
...
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      version: 0.23.9
+    - name: kubernetes
+      version: 1.16.9
+    - name: containerlinux
+      version: 2345.3.1
+    - name: calico
+      version: 3.10.1
```
Comment (cokiengchiara) [inline aws.yaml:467]: Are we upgrading calico to 3.10.4? re: https://github.com/giantswarm/giantswarm/issues/11307#issue-629063741

Given what you said, "This proves it's not enough to change calico version metadata in release definition for that version to be actually used." cc @MartaD @MarcelMue

Comment (sslavic) [reply aws.yaml:467]: Not in this release

File: release-notes/aws/v9.3.3.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# :zap: Giant Swarm Release v9.3.3 for AWS :zap:
+
+**If you are upgrading from 9.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release improves the reliability of NGINX Ingress Controller. Most importantly, liveness probe configuration has been adjusted to be more fault tollerant than readiness probe. This helps shed excess load when the load goes over the replica capacity.
```
Comment (cokiengchiara) [inline release-notes/aws/v9.3.3.md:5]:
```suggestion
This release improves the reliability of NGINX Ingress Controller.

Specifically, liveness probe is configured to be more fault tolerant than readiness probe. This helps shed load when it goes beyond replica capacity, speeding up recovery when NGINX gets overloaded.
```

@sslavic do you think my suggestion makes sense? I want to make the connection with this [PM](https://github.com/giantswarm/giantswarm/issues/11182) clear. That it's about "trying to find a way to maintain some capacity and speed up recovery (without container being restarted by kubelet), while nginx is overloaded."

**Comment by sslavic** [comment]
Tested on giraffe (China)
- 9.3.3 cluster creation
- upgrade from 9.3.2 to 9.3.3

All went well.

Release notes suggestions have been included.

@cokiengchiara please let me know once announcement is ready.

**Comment by cokiengchiara** [comment]
> @cokiengchiara please let me know once announcement is ready.

Ready. Let's do it!

**Pull request merged**